apps/web/components/tools-menu.tsx

```
@@ -0,0 +1,153 @@
+                                <DialogContent ariaTitle="Add Tool" className='max-w-[300px]'>
+                                        <DialogHeader>
+                                                <DialogTitle>Add MCP Tool</DialogTitle>
+                                        </DialogHeader>
+                                        <DialogDescription>
+                                                Add a new tool to the chat.
+                                        </DialogDescription>
+                                        {
+                                                mcpConfig && Object.keys(mcpConfig).length > 0 && (
+
+                                                        Object.keys(mcpConfig).map(key => (
+                                                                <div className='flex flex-row items-center gap-2 divide-x-2 divide-border border-border border rounded-md py-1.5 px-2.5 w-full'>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)